src/cpp/r/session/REmbeddedPosix.cpp

```diff
@@ -228,22 +228,13 @@ bool polledEventHandlerInitialized()
 
 void processEvents()
 {
-#ifdef __APPLE__
```

Comment:
This Apple vs. non-Apple distinction is no longer necessary?

Reply:
I originally thought so, but I decided to back this out. I'm somewhat surprised that, in a function called `processEvents()`, we only ever actually call `R_ProcessEvents()` on macOS. It looks like `R_ProcessEvents` calls a function pointer `ptr_R_ProcessEvents` that's only ever set on macOS, so perhaps that's intentional, but it was odd to see.

I'm going to add a comment anyhow, just to document this.